app/controllers/nominees_controller.rb

```diff
@@ -5,24 +5,26 @@ class NomineesController < ApplicationController
   def create
     @nominee = Nominee.new(nominee_params)
     if @nominee.save
-      redirect_to nominees_path
+      redirect_to policy_nominees_path
     else
       render :new
     end 
   end
 
   def new
-    @nominee = Nominee.new
+    policy = Policy.find_by(params[:id])
```

Comment:
@VaibhaviJoshi-BTC : You are creating nominee after creating policy , but it should be created while creating policy